pkg/controller/case_test.go

```diff
@@ -0,0 +1,583 @@
+/*
```

Comment:
IMO we should add some information/table which will show us what kind of test it covers.

Right now it's hard to me check what test was already replaced with the new approach.

Reply:
I've created a table in the issue: https://github.com/kyma-project/kyma/issues/2840